core/src/datayoga_core/__init__.py

```diff
@@ -1,6 +1,24 @@
 import logging
 from typing import Any, Dict, List, Optional
 
+try:
```

Comment:
@vd2org This should belong in the write.relational block. the main datayoga.core should be agnostic to what the blocks do.

Reply:
and the snippet can be moved to the utils in relational block folder (so can be reused with the test)

Reply:
The same as above.

Reply:
why? I would assume it just needs to happen before the oracle import happens the first time. which can happen in the block init. can you explain?